client/index.html

```diff
@@ -274,7 +274,7 @@ <h1 class="title">Settings</h1>
 								<label for="theme-select" class="sr-only">Theme</label>
 								<select id="theme-select" name="theme" class="input">
 									{{#each themes}}
-										<option value="{{filename}}">
+										<option value="{{url}}">
```

Comment:
Fixed this now. All themes use `/theme/themeName.css` and we still use the name of the theme for the value, so this should work now.